addons/binding/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/api/PowerFlowRealtimeSite.java

```diff
@@ -42,6 +42,9 @@
     private String meterLocation;
 
     public String getMode() {
+        if (mode == null) {
+            mode = new String();
```

Comment:
Better initialize with empty string: `mode = ""`. like you've done at other places.